packages/design-tokens/bin/buildCssVarTokens.ts

```diff
@@ -99,6 +106,8 @@ const run = () => {
     path.resolve(jsonOutput, "color.json"),
     formatJson(
       JSON.stringify({
+        color: customPropertiesTheme.color,
+        DEPRECATED: customPropertiesTheme.DEPRECATED,
```

Comment:
Since we're planning to do a breaking change in a subsequent PR, let's discuss ripping the bandaid off for the deprecated tokens while we do it.

Reply:
Beat you to it in the draft hehe
https://github.com/cultureamp/kaizen-design-system/pull/1760

Reply:
Snap